resources/views/checkout/partials/sidebar/totals.blade.php

```diff
@@ -1,26 +1,27 @@
 <div>
```

Comment:
We can't place the `v-cloak` on the parent?

Reply:
I only used v-cloaks on the elements that are loaded with js. Some elements within this component can already be visible like titles. We could place it on the parent, i just prefer to put it only on the elements that are loaded with a delay